kfp-tekton/base/kfp-tekton.yaml

```diff
@@ -0,0 +1,3172 @@
+        env:
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+        imagePullPolicy: Always
+        name: driver-registrar
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /csi
+          name: socket-dir
+        - mountPath: /registration
+          name: registration-dir
+      - args:
+        - --v=5
+        - --endpoint=$(CSI_ENDPOINT)
+        - --nodeid=$(KUBE_NODE_NAME)
+        env:
+        - name: CSI_ENDPOINT
+          value: unix:///csi/csi.sock
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        - name: cheap
+          value: "off"
+        image: quay.io/datashim/csi-s3:latest-amd64
+        imagePullPolicy: Always
+        name: kfp-csi-s3
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /csi
+          name: socket-dir
+        - mountPath: /var/data/kubelet/pods
+          mountPropagation: Bidirectional
+          name: mountpoint-dir
+        - mountPath: /dev
+          name: dev-dir
+      serviceAccountName: kfp-csi-s3
+      volumes:
+      - hostPath:
+          path: /var/data/kubelet/plugins/kfp-csi-s3
+          type: DirectoryOrCreate
+        name: socket-dir
+      - hostPath:
+          path: /var/data/kubelet/pods
+          type: DirectoryOrCreate
+        name: mountpoint-dir
+      - hostPath:
+          path: /var/data/kubelet/plugins_registry
+          type: Directory
+        name: registration-dir
+      - hostPath:
+          path: /dev
+          type: Directory
+        name: dev-dir
+---
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  labels:
+    application-crd-id: kubeflow-pipelines
+  name: ch.ctrox.csi.s3-driver
+spec:
+  attachRequired: false
+  podInfoOnMount: false
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipeline-loops
+    pipeline.tekton.dev/release: devel
+  name: webhook.pipelineloop.custom.tekton.dev
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
+  clientConfig:
+    service:
+      name: tekton-pipelineloop-webhook
+      namespace: openshift-pipelines
+  failurePolicy: Fail
+  name: webhook.pipelineloop.custom.tekton.dev
+  sideEffects: None
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipeline-loops
+    pipeline.tekton.dev/release: devel
+  name: validation.webhook.pipelineloop.custom.tekton.dev
+webhooks:
```

Comment:
There's no `rules` file defined here...so it seems like...it does nothing ([docs](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-rules))? Same with the `MutatingWebhookConfiguration`. I'm not super familiar with admission webhooks, I think @tumido or @harshad16 might know. 

If it does nothing...we could probably scrap the deployment/service associated with this as well..